agents-and-function-calling/bedrock-agents/features-examples/13-create-agent-using-CDK/lambda/collections/index.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+import os
+import time
+import boto3
+from urllib import parse
+from requests_aws4auth import AWS4Auth
+from opensearchpy import OpenSearch, RequestsHttpConnection
+
+
+def create_collection_index(host: str, index_name: str, metadata_field_name: str, text_field_name: str,
+                            vector_field_name: str, vector_size: int = 1024):
+    """
+    Create an index in the given collection with the given param
+    
+    Parameters
+    ----------
+    host : endpoint for OpenSearch Collection
+    index_name : name of the index to create
+    metadata_field_name: name of the metadata field
+    text_field_name: name of the text field
+    vector_field_name : name of the vector field
+    vector_size : Dimension of the vector. Depends on the embeddings model used. Check:
+                       https://docs.aws.amazon.com/bedrock/latest/userguide/knowledge-base-setup.html
+    """
+    credentials = boto3.Session().get_credentials()
+    awsauth = AWS4Auth(credentials.access_key, credentials.secret_key,
```

Comment:
could we follow PEP8 indentation?

Reply:
Run pep8 on full file and did some manual indentation to look good.